spec/index.bs

```diff
@@ -1589,6 +1589,11 @@ applications should generally request the "worst" limits that work for their con
         {{GPURenderPassDescriptor}}.{{GPURenderPassDescriptor/colorAttachments}},
         and {{GPURenderPassLayout}}.{{GPURenderPassLayout/colorFormats}}.
 
+    <tr><td><dfn>maxTotalColorAttachmentDepth</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>32
+    <tr class=row-continuation><td colspan=4>
+        The maximum number of bytes necessary to hold a pixel, across all color attachments.
```

Comment:
I'd use "fragment" or "sample" instead of "pixel" so the meaning is clearer in the multisampled case.

Reply:
Right, as above, I think "pixel" is actually the term-of-art to refer to "the thing that includes all samples in all relevant texels of the render targets."

Reply:
Hmm. The accounting algorithm doesn't multiply by the sampleCount or anything like that though. Isn't this effectively the size of all of the data coming out of the fragment shader (plus blending etc)?